stories/Components/TableStoriesDocs/TableDocs.mdx

```diff
@@ -69,11 +86,15 @@ const columnData = [
 ];
 ```
 
-- The `ellipsis` key is used to truncate any large text that overflows the given width for a cell.
-- `defaultPageSize` by default is set to `100`.
-- `onRowClick` handler returns `selectedRowKey and selectedRow`.
-- `onRowSelect` handler returns `selectedRowKeys and selectedRows`.
-- `handlePageChange` returns `page and pageSize`.
+Keys for a column definition:
+- `title` : The title of a column. Accepts a react Node.
```

Comment:
```suggestion
- `title` : The title of a column. Accepts a React Node(JSX) or a string.
```

Reply:
One thing I've missed here is that `title` also accepts a method that returns a ReactNode. I'll be adding that too.